src/client/debugger/jupyter/kernelDebugAdapter.ts

```diff
@@ -256,6 +257,21 @@ export class KernelDebugAdapter implements DebugAdapter, IKernelDebugAdapter, ID
         });
     }
 
+    private lookupCellByLongName(sourcePath: string): NotebookCell | undefined {
+        if (!this.platformService.isWindows) {
+            return undefined;
+        }
+
+        sourcePath = path.normalize(sourcePath);
+        for (let [file, cell] of this.fileToCell.entries()) {
+            if (isShortNamePath(file) && shortNameMatchesLongName(file, sourcePath)) {
```

Comment:
Oh so we only have issues with short names for cell files.
I thought we had issues with all paths in general.

Reply:
Where else would you get paths from the kernel? Definitely could be other issues too. I didn't look for anything else.